### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by antony**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: packages/create-svelte/templates/default/package.template.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "~TODO~",
 	"version": "0.0.1",
+	"private": true,
```
Comment (dominikg) [inline packages/create-svelte/templates/default/package.template.json:4]: should we do this in a separate PR ? .template.json won't be found by changesets

File: packages/create-svelte/templates/skeleton/package.template.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "~TODO~",
 	"version": "0.0.1",
+	"private": true,
```
Comment (dominikg) [inline packages/create-svelte/templates/skeleton/package.template.json:4]: see above

File: packages/create-svelte/templates/default/package.template.json (hunk 1 above)
Comment (antony) [reply packages/create-svelte/templates/default/package.template.json:4]: good point, I hadn't spotted this. Agreed.

**Review by dominikg — APPROVED** [review]
